src/Stryker.CLI/Stryker.CLI/FileBasedInput.cs

```diff
@@ -90,6 +90,9 @@ public class FileBasedInput : IExtraData
 
         [JsonExtensionData]
         public Dictionary<string, JsonElement> ExtraData { get; init; }
+
+        [JsonPropertyName("tests-failure-break")]
+        public bool? TestsFailureBreak { get; init; }
```

Comment:
Is there a good usecase for having the config option in the config file?

Reply:
We relay on config files in our pipelines, so it is crucial for me :)

Reply:
Our philosophy is this:

Config file options are those options that rarely change for a project. They are the defaults necessary for correctly running stryker on the project within the context of the team using it. Commandline options are those options that are used most often in one-off scenarios. Configuration you need to overwrite from it's default depending on the current use case (CI vs developer machine for example). To limit configuration overload (especially on the commandline) we prefer to only enable configuration options where they fit with the above definitions.

With this explanation do you believe that the option is needed both in the configfile and on the commandline? I can see a use for having it both in the configfile and on the commandline but if you don't then I would prefer to only add it where you would need it. We can always add it later.

Reply:
But it is up to you to decide ofc

Reply:
Ok then lets leave only CLI option